File: sorting in Arrays/Python/bubble_sort.py
class BubbleSort:
    def sortArray(self, arr):
        for i in range(len(arr)):
            for j in range(len(arr)):
                if arr[j] > arr[i]: # arr[j] < arr[i] for descending order
                    arr[i], arr[j] = arr[j], arr[i]
        

        


if __name__ == "__main__":
    arr = [2, 4, 1, 3, 5]
    BubbleSort().sortArray(arr)
    print(arr)

File: sorting in Arrays/JAVA/BubbleSort.java
public class BubbleSort {
    public void sortArray(int[] arr){
        for(int i = 0; i < arr.length; i++){
            for(int j = 0; j < arr.length - i - 1; j++){
                if(arr[j] > arr[j+1]){ /*arr[j]<arr[i] for descending order*/
                    int temp = arr[j+1];
                    arr[j+1] = arr[j];
                    arr[j] = temp;

                }
            }
        }
    }
    public static void main(String[] args) {
        BubbleSort bubbleSort = new BubbleSort();
        
        int[] arr = {2, 4, 1, 3, 5};
        bubbleSort.sortArray(arr);

        for(int i=0; i< arr.length; i++){
            System.out.println(arr[i]);
        }
        
    }
}
